pkg/controller/deliverable/reconciler_test.go

```diff
@@ -261,19 +261,50 @@ var _ = Describe("Reconciler", func() {
 			Expect(resourceRealizer).To(Equal(builtResourceRealizer))
```

Comment:
I was wondering why this assert wasn't in the context of the service account specified in the delivery or when using the default service account. Should this be a separate "It" block?

Reply:
This makes sense to me. Addressed in latest commit.